modules/mediawiki/files/nginx/mediawiki.conf

```diff
@@ -63,6 +63,30 @@ server {
 	include /etc/nginx/mediawiki-includes;
 }
 
+
+server {
+	listen [::]:443 ssl spdy;
+
+	server_name boulderwiki.org;
+	root /srv/mediawiki;
+
+	ssl_certificate /etc/ssl/certs/boulderwiki.org.crt;
+	ssl_certificate_key /etc/ssl/private/boulderwiki.org.key;
+
+	ssl_trusted_certificate /etc/ssl/certs/StartSSL.crt;
+
+	add_header Strict-Transport-Security "max-age=2419200";
+
+	include /etc/nginx/mediawiki-includes;
+}
+
+server {
+	# Redirect www.boulderwiki to straight boulderwiki
+	listen [::]:443 ssl spdy;
+	server_name www.boulderwiki.org;
+	return 301 https://boulderwiki.org$request_uri;
```

Comment:
@Southparkfan I'm not sure what you mean. Are you talking about this location directive?

```
 	location ~ ^(.*)$ {
 		return 301 https://private.revi.wiki$1;
 	}
```

It seems unnecessary to force a regex match to all urls if it's just going to do the same thing for all urls. In my testing elsewhere, this redirect works as expected.